src/datasources/cache/cache.router.ts

```diff
@@ -407,7 +408,8 @@ export class CacheRouter {
   }
 
   static getRelayKey(args: { chainId: string; address: string }): string {
-    return `${args.chainId}_${CacheRouter.RELAY_KEY}_${args.address}`;
+    // Ensure address is checksummed to always have a consistent cache key
+    return `${args.chainId}_${CacheRouter.RELAY_KEY}_${getAddress(args.address)}`;
```

Comment:
Missed this one: I think this should be outside of the scope of this PR. Also, at this level we should be as close as possible to Redis primitives. Setting the key as a checksummed address should happen in outer layers (that handle more ethereum related logic).

Reply:
Agreed. This is a remnant from an older change that shouldn't be there. I manually removed it in 7c9718a097a267c29733988e72fb26f7f5971361.